app/modules/video-players/jwplayer-video-ads.js

```diff
@@ -0,0 +1,38 @@
+import Ads from '../ads';
+
+export default class JWPlayerVideoAds {
+	constructor(params) {
+		this.params = params;
+	}
+
+	getConfig() {
+		if (this.params.noAds) {
+			return this.params;
+		} else if (this.isA9VideoEnabled()) {
+			return this.parseBidderParameters()
+				.catch(() => {});
+		} else {
+			return {};
+		}
+	}
+
+	parseBidderParameters() {
+		const a9 = Ads.getInstance().a9;
+
+		if (!a9) {
+			return {};
```

Comment:
Return points seem inconsistent -> here we return object and below we return a promise. Could we unify to e.g. always return a promise?

Reply:
it's consumed by parent promise anyway, but sure, it'll be more readable 👍